lib/librato/metrics/smart_json.rb

```diff
@@ -4,18 +4,38 @@ class SmartJSON
       extend SingleForwardable
 
       if defined?(::MultiJson)
-        # MultiJSON >= 1.3.0
-        if MultiJson.respond_to?(:load)
-          def_delegator MultiJson, :load, :read
-        else
-          def_delegator MultiJson, :decode, :read
-        end
+        if RUBY_VERSION <= "2.3.0"
```

Comment:
@nextmat for readability, does it make sense to just wrap these methods regardless of version? looks like we switched to delegation in `v1.0.2`.